ariadne/transformations.py

```diff
@@ -210,6 +192,7 @@ class BaseCoordConverter(BaseTransformer):
 
     def __init__(self, convert_function, drop_old=False, from_columns=('x', 'y'),
                  to_columns=('r', 'phi'), postfix='general_convert'):
+        super().__init__(drop_old=drop_old, columns=list(to_columns) + ['z'])
```

Comment:
why not to define `from_columns = ('x', 'y', 'z')`. I don't understand the purpose of this line `columns=list(to_columns) + ['z']
`

Reply:
logic was to use only realy converted columns, not all. Now i fixed it, it is realy nonlogical